gluefactory/models/lines/pold2_extractor.py

```diff
@@ -18,11 +18,13 @@
 from functools import cmp_to_key
 import matplotlib.pyplot as plt
 from omegaconf import OmegaConf
+from tqdm import tqdm
 
 from gluefactory.settings import root
 from gluefactory.models.lines.pold2_mlp import POLD2_MLP
 from gluefactory.models.base_model import BaseModel
 from gluefactory.settings import DATA_PATH
+from gluefactory.models.lines.line_refinement import merge_lines_torch
```

Comment:
Added comment for clarity: the method is not used in the extractor currently. It will be built in in another refactoring PR